pkg/hub/controllers/placemanifestworkcontroller/placemanifestwork_deploy_reconcile.go

```diff
@@ -18,5 +27,92 @@ type deployReconciler struct {
 
 func (d *deployReconciler) reconcile(ctx context.Context, pw *workapiv1alpha1.PlaceManifestWork) (*workapiv1alpha1.PlaceManifestWork, reconcileState, error) {
 	// TODO the main manifestwork create/update/delete logic, we may also need to manage the status condition for whether manifestwork is created here
+	klog.Info("reconciling placeManifestWork ", pw.Name)
+
+	placement, err := d.placementLister.Placements(pw.Namespace).Get(pw.Spec.PlacementRef.Name)
+	if err != nil {
```

Comment:
should treat NotFoundError differently. So we set condition if it is not found and return reconcileStop and nil error, and return error directly with reconcileStop if it is not NotFoundError. e.g.

```go
switch {
case errors.IsNotFoundError:
    // set condition
   return pw, reconcileStop, nil
case error != nil:
   return pw, reconcileStop, err
}
```

Reply:
done